submissions/DaniaB24/js/app.js

```diff
@@ -0,0 +1,120 @@
+const cardsOfFriends = document.getElementById("allHumans");
+const searchBar = document.getElementById("search");
+
+let filteredFriends;
+let allFriends;
+let sortedFriends;
+let friends;
+fetch("https://randomuser.me/api/?results=10")
+  .then((response) => response.json())
+  .then((users) => {
+    friends = users.results;
+    renderFriends(friends);
+    displayUsers(friends);
+    document.addEventListener("click", (event) =>
+      filterSetting(event, friends)
+    );
+  });
+
+function filterSetting(event, friends) {
+  renderFriends(friends);
+  filterByGender(friends);
+
+  if (event.target.closest(".filter_menu"))
+    sortFriends(event.target, filteredFriends);
```

Comment:
Looks like you're missing brackets here. So `if` is useless and you always do the sorting.

It's not a python where you can use spaces. :) So please, use prettier.

Reply:
ohhh, thanks a lot:) it helped me fix the bug